package.json

```diff
@@ -42,6 +42,9 @@
         "aws-cdk-lib": "^2.0.0",
         "constructs": "^10.0.43"
     },
+    "optionalDependencies": {
+        "@ziglang/cli": "^0.0.8"
```

Comment:
The `--save-optional` along with `@ziglang/cli` is a neat idea! I wonder why I didn't think of that.